test/behavioral/behavioral.ts

```diff
@@ -1,4 +1,4 @@
-import {$, Snappit, IConfig, WebDriver, By} from "snappit-visual-regression";
+import {$, Snappit, IConfig, WebDriver, By, WebElement} from "snappit-visual-regression";
```

Comment:
_(outside current scope of work)_
This file needs refactored, badly.

* If kept, this file should act as an entrypoint for behavioral testing (including/launching individual files for each component's behavioral tests).
* Behavioral tests should be organized so that each file tests a single component. Currently, we're testing Tabs, Nav, and Reveals (via `<hx-disclosure>` interaction) in a single file.
* The "nav" tests should test against the controlled demo in `/components/navigation` instead of the variable left nav.